### PR TITLE
Add Validator table and handle create validator message from genesis

### PR DESCRIPTION
**Pull request by Benzbeeb**
Add Validator table and handle create validator message from genesis file and tx for emitter and flusher

Fixed: #2109

## Implementation details
1. Add validators table to cdb.
2. Add new_validator handler for flusher.
3. Add genesis validator from genesis state to Kafka.
4. Implement handle msgCreateValidator.

---

Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: flusher/flusher/db.py [hunk 1]
```diff
@@ -148,3 +148,23 @@ def Column(*args, **kwargs):
     ),
 )
 
+validators = sa.Table(
+    "validators",
+    metadata,
+    Column("operator_address", sa.String, primary_key=True),
+    Column("consensus_address", sa.String, unique=True),
```
Comment (taobun) [inline flusher/flusher/db.py:177]: Add fk

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]

File: chain/emitter/staking.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		"moniker":               val.Description.Moniker,
+		"identity":              val.Description.Identity,
+		"website":               val.Description.Website,
+		"details":               val.Description.Details,
+		"commission_rate":       val.Commission.Rate.String(),
+		"commission_max_rate":   val.Commission.MaxRate.String(),
+		"commission_max_change": val.Commission.MaxChangeRate.String(),
+		"min_self_delegation":   val.MinSelfDelegation.String(),
+		"tokens":                val.Tokens.Uint64(),
+		"jailed":                false,
```
Comment (sorawit) [inline chain/emitter/staking.go]: Should we read this from `val`?

File: chain/emitter/staking.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package emitter
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+)
+
+func (app *App) SetValidator(addrs sdk.ValAddress, blockHeight int64) {
```
Comment (sorawit) [inline chain/emitter/staking.go]: Let's call this `emitSetValidator`

File: chain/emitter/staking.go [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	val, _ := app.StakingKeeper.GetValidator(app.DeliverContext, addrs)
```
Comment (sorawit) [inline chain/emitter/staking.go:9]: Let's remove `blockHeight` argument.

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]

**Pull request merged**